app/controllers/patients_controller.rb

```diff
@@ -89,6 +89,18 @@ def data_entry_create
     end
   end
 
+  # allow ONLY admins to delete patients, and only then if patient does not have pledge_sent
+  def destroy
+    @patient = Patient.find params[:patient_id]
```

Comment:
There's a `before_action` we can take advantage of here -- on L4, `before_action :find_patient, only: [:edit, :update, :download]` basically does this! Add `:destroy` to the array and it'll do the same thing, pretty much, but keep this method clean.